Automattic/jetpack
# VideoPress: re-write video chapters block in TS

This is a janitorial PR:
* Re-write part of the Video Chapters block code in TypeScript

<img width="534" alt="image" src="https://user-images.githubusercontent.com/77539/208298554-106e4bb7-9c4b-41ed-9f9a-c90926fd99a8.png">

<img width="991" alt="Screen Shot 2022-12-18 at 12 33 33" src="https://user-images.githubusercontent.com/77539/208298602-084bccb1-3b9e-4712-89bb-bbbba2bd6b0f.png">

* Add missing `@wordpress/dom-ready` dependency
* Remove unused editor.js file

Part of https://github.com/Automattic/jetpack/issues/27867

#### Changes proposed in this Pull Request:

* VideoPress: re-write video chapters block in TS

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Enable beta extensions
* Go to block editor
* Insert the Video Chapters block
* Confirm it should work as expected. No significant changes here.

<img width="650" alt="image" src="https://user-images.githubusercontent.com/77539/206903220-69ad8019-aa20-4f3c-8dcb-37adab912307.png">
